languages/swift/exercises/concept/functions/.docs/after.md

```diff
@@ -0,0 +1,49 @@
+Great job! Functions are at the heart of Swift programming, so understanding them is vital to becoming fluent in Swift.
+
+## Key Takeaways from this Exercise
```

Comment:
This "Key Takeaways" format is different from the other Swift exercise's `after.md` files. Is that intentional? It might be worth standardizing on one format.

Reply:
Yes, this was a question that had arisen earlier. There was some concern that the after looking so similar to the introduction might lead the student to skim it and miss the new information. I had seen this style of after in IIRC the Elixir track and wanted other eyeballs to look at it and try to see which was better. I personally prefer the style that is similar to the introduction.md.

Reply:
Yes me too. It is also worth to note that we will make it so that students will probably want to revisit these after.md files later on, as a sort of reference, at which point they'll probably don't even remember the introduction.md document.